plugins/httpdispatcher.koplugin/main.lua

```diff
@@ -0,0 +1,264 @@
+--[[--
+This plugin allows for remotely triggering KOReader events via HTTP.
+
+Refer to frontend/dispatcher.lua for the list of events supported.
```

Comment:
From what I understand, your plugin is not limited to the event listed in dispatcher.lua, which is good !
So, the wording should be "for some examples of events supported" :)

Reply:
Do we have a complete list of all the events supported? Perhaps I could include them as well for future reference.

Reply:
There will be one :) https://github.com/koreader/koreader/pull/10328#issuecomment-1546692053

Reply:
@zwim has been working on that in #10328 ;).